src/main/java/amazin/controller/ApplicationController.java

```diff
@@ -1,18 +1,19 @@
-package amazin;
+package amazin.controller;
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
+import amazin.repository.BookRepository;
+
 @Controller
 public class ApplicationController {
 
     @Autowired
     BookRepository repository;
```

Comment:
I would rename this bookRepository, in case we need to include additional repositories in the future.

Reply:
good point. This has now been renamed.